owl/EASE-SAY.owl

```diff
@@ -113,6 +244,83 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#ContainmentTheory -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#ContainmentTheory">
```

Comment:
I think role axioms are missing.
**containee** and **container** seem to be not defined in *ControlTheory*, but rather locatum and relatum